coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -1636,7 +1633,6 @@ def forward(self, input_data, weights):
                 "kernel_size",
                 "stride",
                 "padding",
-                "dilation",
```

Comment:
`diliation` value is not substituted.
`diliation` is not supported in dynamic convolution.

Reply:
You're right this is not supported:
https://github.com/apple/coremltools/blob/d9215d6f3ebda56e44f406ce1142b2809489816f/coremltools/converters/mil/mil/ops/defs/iOS15/conv.py#L182-L183